synapse/handlers/federation.py

```diff
@@ -1046,7 +1265,36 @@ async def on_backfill_request(
         # Synapse asks for 100 events per backfill request. Do not allow more.
         limit = min(limit, 100)
 
-        events = await self.store.get_backfill_events(room_id, pdu_list, limit)
+        # events = await self.store.get_backfill_events(room_id, pdu_list, limit)
```

Comment:
TODO: Remove unused `store.get_backfill_events` now that we have our own `get_backfill_events` in the handler here